js/packages/phoenix-client/examples/dataset_and_experiments_quickstart.ts

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable no-console */
+import { OpenAI } from "openai";
+import { createClient } from "../src";
+import { runExperiment, asEvaluator } from "../src/experiments";
+import { AnnotatorKind } from "../src/types/annotations";
+import { Example } from "../src/types/datasets";
+import { createDataset } from "../src/datasets/createDataset";
+
+// Replace with your actual OpenAI API key
+const OPENAI_API_KEY = process.env.OPENAI_API_KEY;
+
+if (!OPENAI_API_KEY) {
+  console.error("Please set the OPENAI_API_KEY environment variable");
+  process.exit(1);
+}
+
+const openai = new OpenAI({
+  apiKey: OPENAI_API_KEY,
+});
+
+// Initialize Phoenix client
+const client = createClient();
+
+// Launch Phoenix app in browser - Note: In a script, you would need to use a web server
+// or the user would need to open the Phoenix UI separately
+console.log("Access Phoenix UI at http://localhost:6006");
+
+async function main() {
+  // First, create a dataset via the API
+  console.log("Creating dataset...");
+
+  // Create examples without id field since it's not expected in the Example type
+  const examples = [
+    {
+      input: { question: "What is Paul Graham known for?" },
+      output: {
+        answer:
+          "Co-founding Y Combinator and writing on startups and techology.",
+      },
+      metadata: { topic: "tech" },
+    },
+    {
+      input: { question: "What companies did Elon Musk found?" },
+      output: {
+        answer:
+          "Tesla, SpaceX, Neuralink, The Boring Company, and co-founded PayPal.",
+      },
+      metadata: { topic: "entrepreneurs" },
+    },
+    {
+      input: { question: "What is Moore's Law?" },
+      output: {
+        answer:
+          "The observation that the number of transistors in a dense integrated circuit doubles about every two years.",
+      },
+      metadata: { topic: "computing" },
+    },
+  ] as Example[];
```

Comment:
We should avoid `as` when we can, its more dangerous than `satisfies` or an annotation `examples: Example[]`.

```suggestion
  ] satisfies Example[];
```